hudi-common/src/main/avro/HoodieCommitMetadata.avsc

```diff
@@ -129,6 +129,11 @@
          }],
          "default": null
       },
+      {
```

Comment:
can we add this at the end? is nt that need for this to be backwards compatible?

Reply:
> is nt that need for this to be backwards compatible

Move it to the end. IMO, it has no impact on compatibility.

Reply:
Agree with Vinoth, Avro schema evolution expects new fields to be appended.